hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/UpdatingEntryProcessor.java

```diff
@@ -63,11 +66,13 @@ private UpdatingEntryProcessor() {
     private UpdatingEntryProcessor(
             KvRowProjector.Supplier rowProjectorSupplier,
             Projector.Supplier valueProjectorSupplier,
-            List<Object> arguments
+            List<Object> arguments,
+            HazelcastInstance hzInstance
```

Comment:
can we use `HazelcastInstanceAware` here?

Reply:
I am really not sure, that's why I passed it as ctor argument